File: in_Java/1184 Distance Between Bus Stops.java
class Solution {
    public int distanceBetweenBusStops(int[] distance, int start, int destination) {
        int n = 0;
        int nn = 0;
        for(int i : distance) n += i;
        int a = Math.min(start,destination);
        int b = Math.max(start, destination);
        for(int i = a; i < b; i++){
            nn += distance[i];
        }
        return Math.min(nn, n - nn);
    }
}

File: in_Python/1184 Distance Between Bus Stops.py
class Solution(object):
    def distanceBetweenBusStops(self, distance, start, destination):
        """
        :type distance: List[int]
        :type start: int
        :type destination: int
        :rtype: int
        """
        ss = 0
        for i in distance:
            ss += i
        s = 0
        a = min(start,destination)
        b = max(start,destination)
        for i in range(a,b):
            s += distance[i]
        return min(s, ss-s)